### PR TITLE
Update README.md

### LINKED ISSUE #14: add readme badges
I want to add readme badges.Please assign this issue to me under GSSOC.

### PR DESCRIPTION
**Pull request by S-ishita**
Closes #14 
Screenshot from 2023-05-21 23-07-09

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,16 @@
 # [Resource gallery🛸](https://resourcegallery.live/)
+
+<div align="center">
+<img src="https://forthebadge.com/images/badges/built-with-love.svg" />
+<img src="https://forthebadge.com/images/badges/uses-brains.svg" />
```
Comment (Astrodevil) [inline README.md:5]: change uses-astro
&
remove responsibility badge

Comment (S-ishita) [reply README.md:5]: This cant be customized..other standard templates are there..such as powered by coffee,powered by pull requests etc.Do you want me to remove it?

Comment (Astrodevil) [reply README.md:5]: which generator you used?

Comment (S-ishita) [reply README.md:5]: For the badge

Comment (Astrodevil) [reply README.md:5]: You can generate custom one too from there. try to generate and add here

**Review by bhartik021 — APPROVED** [review]

**Comment by Astrodevil** [comment]
Hey @S-ishita Please do change things as I asked earlier. raise new PR, points for both will be given for this PR only

**Pull request merged**